Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -138,6 +140,185 @@ public class PushDeploymentController : ApiController
             }
         }
 
+        [HttpPost]
+        [HttpPut]
+        public async Task<HttpResponseMessage> OnePushDeploy(
+            [FromUri] bool isAsync = false,
+            [FromUri] string author = null,
+            [FromUri] string authorEmail = null,
+            [FromUri] string deployer = OneDeploy,
```

Comment:
>deployer [](start = 29, length = 8)

What happens if the client does:
POST /api/onedeploy?type=war&deployer=onedeploytest

Do we want this to be customisable? If not, remove it from the query parameter list.